src/styles/_reset.scss

```diff
@@ -0,0 +1,11 @@
+html,
+h2,
+body,
+ul,
+a {
+  font-family: $font-stack;
```

Comment:
assign font family to body only

Reply:
you don't need all those styles to the elements you've written there

Reply:
The reason why I added all of these tags here are next: 
For example, if I would delete h2 tag from here, text inside h2 tag will have default size and I would need to add font-size to the class which customizes h2 tag.
Also, I reset text-decoration for the "a" tag, and list-style for the ul, for not to write inside their classes.
The body tag is here for the margin: 0;
Html tag for font-family and font-size;

Reply:
if you need only margins for body, then write them separately with body, because body may not need other styles

don't assign just for sure, use the styles you really need at this time

Reply:
Also need margin 0 for h2 tag. So, decided to assign margin 0 both for body and h2.